src/components/WebexNoMedia/WebexNoMedia.scss

```diff
@@ -0,0 +1,28 @@
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-no-media {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-size: 0.875rem;
+  color: var(--wxc-text-color);
+
+  .icon-media-issue {
+    margin-bottom: 1.25rem;
+  }
+
+  .title {
+    margin-bottom: 1.5rem;
+    font-size: 1rem;
+    font-family: $brand-font-bold;
+    color: var(--wxc-error-color);
+  }
+
+  P {
+    &:first-of-type {
+      margin-bottom: 1.875rem;
+    }
+
+    span {
```

Comment:
use `<b>` or `<strong>` instead of `<span>`, it seems the purpose is to make that text stronger

Reply:
Doesn't look the same. with strong or b the text rendered by the browser looks blurry.